tests/cram.t/run.t

```
@@ -50,21 +50,21 @@ combinations of produce-models et al.
 First, if model generation is not enabled, we should error out when a
 `(get-model)` statement is issued:
 
-  $ echo '(get-model)' | alt-ergo --frontend dolmen -i smtlib2 -o smtlib2
+  $ echo '(get-model)' | alt-ergo --frontend dolmen -i smtlib2 -o smtlib2 --continue-on-error
```

Why do we need `--continue-on-error` here? Is it because cram tests require a 0 exit code?

---

Yes, if we don't add this option the output is suffixed with `[1]` (the error code); for clarity it is better to just ignore the error.